### ISSUE TITLE
[bug]scarecrow

### ISSUE THREAD

**Issue by nitbe** [MehVahdJukaar/Dummy_1.18#35]
version：1.20.1 forge
mod version：1.20-1.8.19
Animals and mobs in the whitelist cannot be placed in the scarecrow area. It seems that they are not afraid of the scarecrow.
![3RB20{)R4RD`ZXJ@XZ86YDN](https://github.com/user-attachments/assets/5253774c-ae21-47fd-a1d3-78945fdbbfff)
![NZLQO CKC%{P~P_1HI_4T20](https://github.com/user-attachments/assets/419558d4-7f71-45d2-8e98-8424392d1dfb)

**Comment by MehVahdJukaar** [comment]
does this work with animals alreaedy on the tag? aka are you sure you tagged those animals correctly?

**Comment by nitbe** [comment]
Looks like it, I thought it defaulted to all friendly creatures

**Comment by nitbe** [comment]
I've tested it,  can't intimidation the crows in alexmobs mod, which is weird, even though I've added a whitelist, I've tried adding creatures from other mods, and it works.

**Comment by nitbe** [comment]
![$X927F5O YCXE{AVQ~3`RDJ_tmb](https://github.com/user-attachments/assets/6c313819-3efe-45ea-9e6a-7cd240eabe17)

**Comment by MehVahdJukaar** [comment]
Yeah I got no clue. All the other mobs do work it's just the crows, ironically
